worlds/peaks_of_yore/data.py

```diff
@@ -0,0 +1,169 @@
+class LocationData:
+    name: str
+    type: str
+    id: int
+    region: int
```

Comment:
I'd recommend considering using an IntEnum for your regions instead of magic numbers here